.github/files/.markdownlint.jsonc

```diff
@@ -0,0 +1,12 @@
+{
+    "config": {
+        "default": true,
+        "relative-links": true
+      },
+      // globs expression. See https://github.com/DavidAnson/markdownlint-cli2#command-line
+      "globs": [
+        "**/*.md",
+        "#CHANGELOG.md"
```

Comment:
Need a little bit more research on the globs expression. The `#CHANGELOG.md`  didn't work.